docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     build:
       context: .
       args:
-        - USE_FIXTURES='false'
+        - USE_FIXTURES='true'
```

Comment:
Is there a reason that you are switching this?

Reply:
Cruft. Did not mean to commit that. I'll revert.